mime_types.h

```diff
@@ -0,0 +1,23 @@
+//
+// mime_types.hpp
```

Comment:
Per CS 130 expectations, can we add a comment linking to where this file came from (the Boost docs)?